.github/workflows/test.yaml

```diff
@@ -24,21 +24,23 @@ on:
 
 jobs:
   test:
-    runs-on: ubuntu-22.04
+    runs-on: ubuntu-24.04
     timeout-minutes: 10
 
     strategy:
       fail-fast: false
       matrix:
         include:
-          - python-version: "3.8"
-            pip-install-spec: "jupyterhub==2.* sqlalchemy==1.*"
```

Comment:
I think the ubuntu 24.04 environment doesn't have python 3.8, so I dropped this now that its also reached end of life.